website/content/v1.10/kubernetes-guides/configuration/inlinemanifests.md

```diff
@@ -0,0 +1,176 @@
+
+You define them in the `extraManifests` section of the machine configuration.
+They’re best suited for shared, versioned, or centrally managed resources.
+
+These manifests are pulled directly by the node during configuration.
+If the node doesn’t have network access to the HTTP endpoint hosting the manifest the installation will fail.
+
+Similarly, if the endpoint is down or returns an error, the manifest will not be applied, and the machine configuration will fail as a result.
+
+Here’s how to reference `extraManifests`:
+
+```yaml
+cluster:
+  extraManifests:
+    - "https://raw.githubusercontent.com/example/repo/main/manifest.yaml"
+    - "https://gist.githubusercontent.com/user/gist-id/raw/manifest.yaml"
+```
+
+## Resource Ordering Considerations
+
+You must note that although `inlineManifests` and `extraManifests` are defined as arrays, Talos does not guarantee that the items will be applied in the order they appear.
+
+If your resources depend on one another and must be applied in a specific order, define them in a single manifest entry and separate each resource using the `---` YAML document separator.
+
+This ensures they are parsed and applied in the correct order.
+
+Here’s an example:
+
+```yaml
+cluster:
+  inlineManifests:
+    - name: cilium-install
+      contents: |
+        ---
+        apiVersion: v1
+        kind: ServiceAccount
+        metadata:
+          name: cilium-install
+          namespace: kube-system
+        ---
+        apiVersion: rbac.authorization.k8s.io/v1
+        kind: ClusterRoleBinding
+        metadata:
+          name: cilium-install
+        roleRef:
+          apiGroup: rbac.authorization.k8s.io
+          kind: ClusterRole
+          name: cluster-admin
+        subjects:
+        - kind: ServiceAccount
+          name: cilium-install
+          namespace: kube-system
+
+```
+
+## Example Usecase: Install a GitOps controller with extraManifests
+
+A common use case for `inlineManifests` or `extraManifests` is to install a GitOps controller like Flux or ArgoCD.
+Once the controller is running, it connects to your Git repository and automatically applies the rest of your Kubernetes configuration.
+
+Here's how to install the Flux GitOps controller using an `extraManifest`:
+
+1. Create a patch file named `flux-extra-manifest.yaml` that automatically downloads and applies the Flux installation manifest from GitHub:
+
+    ```shell
+    cat << EOF > flux-extra-manifest.yaml
+    cluster:
+    extraManifests:
+        - "https://github.com/fluxcd/flux2/releases/latest/download/install.yaml"
+    EOF
+    ```
+
+1. Create a `CP_IPS` variable that contains the IP addresses of your control plane nodes:
+
+    ```bash
+    CP_IPS="<control-plane-ip-1>,<control-plane-ip-2>,<control-plane-ip-3>"
+    ```
+
+1. Run this command to export your `TALOSCONFIG` variable.
+You can skip this step if you've already done it:
```

Comment:
same comment as before, are we going to put this to every documentation page?

Reply:
I don't like that its there, but Justin made a point that I should add it, as they cant successfully run this command if they are not in the same directory.  And there really isn't a docs to externally link to.

I guess I should just state that they should export their TALOSCONFIG variable if they haven't done so. And since they have done it a couple times before now, they should know what I mean?